C/tasks/purge_system/purge_system.cpp

```diff
@@ -0,0 +1,168 @@
+/*
+ * Fledge Statistics History
+ *
+ * Copyright (c) 2021 Dianomic Systems
+ *
+ * Released under the Apache 2.0 Licence
+ *
+ * Author: Stefano Simonelli
+ */
+
+#include <purge_system.h>
+#include <logger.h>
+
+#include <cstdarg>     /* va_list, va_start, va_arg, va_end */
+#include <cstdlib>
+#include <thread>
+#include <csignal>
+
+using namespace std;
+
+volatile std::sig_atomic_t signalReceived = 0;
+
+static const string DEFAULT_CONFIG = QUOTE({
+	"retainStatsHistory": {
+		"description": "This is the measure of how long to retain statistics history data for and should be measured in days.",
```

Comment:
I would make this something like "The number of days for which full granularity statistics history is maintained."

Reply:
done